File: backend/src/main/java/ipeps/pwd/wallet/payload/createPayload/AddressCreatePayload.java
package ipeps.pwd.wallet.payload.createPayload;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class AddressCreatePayload {
    private String type;
    private String road;
    private String number;
    private String box;
    private String cp;
    private String town;
    private String country;
}

File: frontend/src/app/modules/address/model/payload/address-create.payload.ts
import { PayloadInterface } from "@shared/model";

export interface AddressCreatePayload extends PayloadInterface {
  type: string;
  road: string;
  number: string;
  box: string;
  cp: string;
  town: string;
  country: string;

  deleted: boolean;
  deletedAt: Date;
  deletedBy: string;
}
